Lesson5/README.md

```diff
@@ -0,0 +1,89 @@
+# Compile and run Azure gateway SDK module and sample application
+This sample accompanies [Compile and run Azure gateway SDK module and sample application](#) lesson. You will use scripts to compile the Azure gateway SDK's module and sample application yourself.
+
+## Prerequisites
+See associate lesson for more information.
+
+## Repository information
+
+```txt
+.
+|- module\                    
+|    |- hello_world\           
+|        |- src\                 // Azure gateway SDK hello_world module source code
+|        |   |- hello_world.c
+|        |   |- hello_world.h
+|        |- build.sh             // build script for hello_world module
+|- sample\
+|    |- hello_world\
+|        |- src\                    // Azure gateway SDK hello_world sample application source code
+|        |   |- hello_world.json
+|        |   |- main.c
+|        |- build.sh                // build script for hello_world sample application
+|- config.json
+|- gulpfile.js
+```
+
+## Compile and run hello_world sample
+
+These steps will guide you compile the Azure gateway SDK's hello_world sample application on your Intel NUC. This sample application creates a gateway from hello_world.json file and use two modules in Azure IoT gateway SDK to log a "hello world" message to a file every 5 seconds.
+
+Please follow the [Compile and run Azure gateway SDK module and sample application](#) for detailed walkthough of the steps below.
+
+1. **`npm install`** - Install the npm packages
+
+2. **`gulp init`** - Intialize configuration files in user's profile folder `%USERPROFILE%\.iot-hub-getting-started`
+
+   * `config-gateway.json`: Configuration for connecting to gateway via SSH.
+   * `config-sensortag.json`: Configuration for the sensor will be connected to gateway, which will be send to your Intel NUC.

```

Comment:
Configuration for the sensor connected to gateway. The file will be sent to your Intel NUC.